File: 프로그래머스/lv1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.py
def solution(n, arr1, arr2):
    answer=[]
    for i in range(n):
        newarr=''
        strarr1 = bin(arr1[i])[2:].zfill(n)
        strarr2 = bin(arr2[i])[2:].zfill(n)
        for j in range(n):
            newarr = newarr+'#' if int(strarr1[j]) or int(strarr2[j]) else newarr+' '
        answer.append(newarr)
    return answer

File: 프로그래머스/lv1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.js
function solution(n, arr1, arr2) {
    const change_binary = (num) => {
        return num.toString(2)
    };
    
    const answer = [];
    
    for(let i=0; i < n; i++){
        answer.push((arr1[i] | arr2[i]).toString(2).padStart(n,'0').replaceAll('1', '#').replaceAll('0', ' '));
    };
    
    return answer;
}

